inmemorycompiler/src/main/java/org/togetherjava/tjbot/javap/Javap.java

```diff
@@ -90,7 +90,7 @@ private Javap() {}
             invokeOptions.add(TEMP_FILE_NAME);
 
             javapTaskRunMethod.invoke(javapTaskInstance,
-                    new Object[] {invokeOptions.toArray(new String[0])});
+                    (Object) invokeOptions.toArray(new String[0]));
```

Comment:
use `toArray(String[]::new)` instead if possible. The cast should also be obsolete since `String[]` is subtype of `Object`.

Reply:
`Method#invoke` accepts an object vararag as the last parameter. If I do not cast to `Object` here, it would spread the array (You'd also get an IJ warning).